modules/openapi-generator/src/main/resources/csharp/libraries/generichost/ClientUtils.mustache

```diff
@@ -386,11 +386,6 @@ using System.Runtime.CompilerServices;
         /// </summary>
         public const string SCHEME = "{{{scheme}}}";
 
-        /// <summary>
-        /// The context path of the API
-        /// </summary>
-        public const string CONTEXT_PATH = "{{contextPath}}";
-
```

Comment:
i think this is a breaking change. 

any chance we can keep it for backward compatibility?

Reply:
Done